src/Microsoft.SymbolManifestGenerator/SymbolManifestGenerator.cs

```diff
@@ -63,8 +63,8 @@ public static bool GenerateManifest(ITracer tracer, DirectoryInfo dir, string ma
                     BasedirRelativePath = basedirRelativePath,
                     SymbolKey = runtimeCorrelatedKey.Index,
                     Sha512 = fileHash,
-                    DebugInformationLevel = Private,
-                    LegacyDebugInformationLevel = Private
+                    DebugInformationLevel = TargetDebugLevel,
+                    LegacyDebugInformationLevel = TargetDebugLevel
```

Comment:
I wonder if this is still needed?

Reply:
It shouldn't be needed if all the places that use this SymbolManifestGenerator package is using a more recent symbol.exe   (more recent than ~May)

Reply:
Removed